feat: add ext.bridge docs (#1222)
Pycord-Development/pycord: docs/ext/bridge/index.rst

.. _discord_ext_bridge:

> ``discord.ext.bridge`` -- A module that bridges prefix commands to slash commands

Reviewer: I think this would be a bit better since the examples use `ctx.respond` which is for application commands, showing that this module is built around that
```suggestion
``discord.ext.bridge`` -- A module that bridges slash commands to prefixed commands
```

===================================================================================

.. versionadded:: 2.0

This module allows using one command callback in order to make both a prefix command and a slash command.

.. note::

>     ``ext.bridge`` requires message content intent to be enabled in order to use it.

Reviewer: ```suggestion
    ``ext.bridge`` requires message content intent to be enabled in order for the prefixed commands to be used.
```
maybe?
